composables/use-reader.ts

```
@@ -14,6 +33,13 @@ export default function () {
 
   const bookFileURLWithCORS = computed(() => {
     const url = new URL(`${config.public.likeCoinAPIEndpoint}/ebook-cors/`)
+    if (!nftClassId.value || nftId.value === undefined) {
+      throw createError({
+        statusCode: 400,
+        message: 'Missing required parameters',
+      })
+    }
+
```

I don't think we should throw error in `computed`, also `nftId` can be undefined in the future for those book that are free to view without owning NFT

---

[🔥 Remove error throwing in bookFileURLWithCORS](https://github.com/likecoin/3ook-com/pull/47/commits/2380f10cbc5533454c9b910788f3bfd51119f3fd)